src/main/java/seedu/address/model/group/tasks/TaskInitializer.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.model.group.tasks;
+
+import java.util.ArrayList;
+import java.util.List;
+import seedu.address.model.group.exceptions.TaskException;
+
+public class TaskInitializer {
+    public static TaskList initializeTasks() throws TaskException {
+        // Create an empty TaskList
+        TaskList taskList = new TaskList();
+
+        List<Task> allTasks = new ArrayList<>();
+
+        // Create tasks for CS2101
+        allTasks.add(new Todo("Upload video of op1", TaskStatus.NOT_DONE, TaskModule.CS2101));
+        allTasks.add(new Todo("Complete peer review for op1", TaskStatus.NOT_DONE, TaskModule.CS2101));
+        allTasks.add(new Deadline("Submit slides for op2", TaskStatus.NOT_DONE, TaskModule.CS2101, "29/10/2023 2359"));
+        allTasks.add(new Deadline("Complete peer review", TaskStatus.NOT_DONE, TaskModule.CS2101, "02/11/2023 2359"));
+        allTasks.add(new Todo("Research on the SCQA framework", TaskStatus.NOT_DONE, TaskModule.CS2101));
+        allTasks.add(new Deadline("plan for op2 ", TaskStatus.NOT_DONE, TaskModule.CS2101, "24/10/2023 2359"));
```

Comment:
Standardise capitalising the first letter of each task
```suggestion
        allTasks.add(new Deadline("Plan for OP2 ", TaskStatus.NOT_DONE, TaskModule.CS2101, "24/10/2023 2359"));
```